app/assets/javascripts/discourse/lib/deprecation-silencer.js

```diff
@@ -1,7 +1,5 @@
 const SILENCED_WARN_PREFIXES = [
   "Setting the `jquery-integration` optional feature flag",
```

Comment:
should this be removed?

Reply:
Unfortunately not yet. We're still setting the flag to 'true', which is deprecated.